src/common/execute.go

```diff
@@ -72,6 +78,18 @@ func RunCommand(cmd *exec.Cmd, dir string, showOutput bool) error {
 
 	wg.Wait()
 
+	if err != nil {
+		if !showOutput {
+			return fmt.Errorf(
+				"Failed to run \n%s\n\nStderr: %s\n\nStdout: %s\n\n%s",
```

Comment:
In future I hope you will write it to the file.

Reply:
See comment above.